### ISSUE TITLE
2024-03-02 公共交通オープンデータ最前線2024

### ISSUE THREAD

**Issue by hfu** [UNopenGIS/7#393]
As introduced by @albertkun in #371. 

https://gtfs2024.peatix.com/ 

>公共交通オープンデータの整備・活用などに関する取組紹介・交流イベントです。

> 近年、GTFSデータなどの公共交通オープンデータ公開が盛んになるとともに、データを開発に活かすエンジニアや情報提供事業者、分析に活かすアナリスト、計画や合意形成に活かす自治体・交通事業者が増えています。[ODD（オープンデータ・デイ）](https://opendataday.org/ja/)におけるシンポジウム開催7年目にあたる今年も、行政、事業者、ITエンジニアなどから、各地の最新の取り組みをご紹介いただきながら、交流を深めます。

> 日時：2024年3月2日（土） 13:00～17:00　（懇親会：18:00～20:00）
場所：東京大学 工学部２号館１階 213号大講義室（本郷キャンパス）
　　　　（懇親会：工学部２号館２階展示室）
オンライン：同時Youtube配信（ハイブリッド開催）URLは参加申込をいただいたメールアドレスに送付いたします。
主催：一般社団法人日本バス情報協会
後援：国土交通省、公益社団法人日本バス協会、一般社団法人社会基盤情報流通推進協議会
参加費：2000円　（懇親会：3000円）

**Comment by hfu** [comment]
Let us know if anyone in our community is joining!

**Comment by asahina820** [comment]
I'll join this event 😆

**Comment by hfu** [comment]
Thank you @asahina820 for sharing your participation. I assign this issue to you. No pressure ;-)

**Comment by hfu** [comment]
@asahina820 いかがでしたか？

特段の課題やフィードバックなどなければ、このイシューは閉じようかなと思っています。

**Comment by asahina820** [comment]
せっかくなので、メモを共有します！

**Comment by asahina820** [comment]
## データ整備

### 秋田県におけるGTFSデータの整備とデジタル化の取り組みについて　秋田県大塚さん

秋田県の乗合バス事業者は3つ、県北、中央、県南
国際教養大学豊田先生が県内全路線のデータ作成済
県主催でデータ作成方法や活用方法を習得できる講座を開催した
作成したGTFSデータをバスターミナルや駅などのデジタルサイネージで発信

### 滋賀県のGTFSデータ整備　滋賀県加藤さん

滋賀県ではバス利用者減少
自治体負担のコミバス負担金が増えているため、コミバス利用者を増やしたい
経路検索に出てこないと使われない。GTFSデータ整備を始めた
Googleにも登録。データ整備後にコミバス路線が使われ、安く、早く着くルートが検索できた

### 福岡県内GTFS-JPデータ整備の新たな展開　九州産業大稲永さん

学生と一緒に活動。アプリ開発やバスロケ、データ分析などができるアプリを開発

### 安芸高田市に人を集めたい！　中電工有安さんと　大学生

中国地方の観光地は南に集中している。北にも来てほしい！
GoogleMapで見ると広島市内から神楽門前湯地村に行けないことになっている
バスデータをGoogleに取り込んで来てもらいたいと考えた
西澤さんツールを使ってGTFSデータを作った。バス停の緯度経度を現地に行って取得した
バス停を巡るとバス停が見づらい、見つけにくいなどの課題があった
作ったデータを安芸高田市に提供し、Googleに申請予定！

### GTFSデータだって楽したい　諸星さん

GTFSデータを手で作るの大変。楽したい！ChatGPT活用したい
市川市のデータを作ってもらったが、検証したらダメだった、、
作らせるには知識が必要、、まだ難しそうだが、今後に期待！

**Comment by asahina820** [comment]
## **バスロケ・業務システム**

### 永井運輸　水野さん

バス運転士から「なぜか下川淵で降りる人が多い」と言われた
Excel帳票を見ると、確かに増えている
コストコ、IKEAができたことが原因だった！

### 北見工業大学におけるGTFS-JPをベースとしたユニバーサルロケ展開　北見工業大学

複数のバスロケを扱うためのシステムを開発した
遅延分析のため、GTFSのリアルタイムアーカイブが欲しい
静的、リアルタイムのGTFSデータを保存できるシステムを開発した
減便による影響を可視化できる！

### GTFS-RTを用いた統合型リアルタイム情報の提供　高知大学片岡さん

GTFSデータ対応Web時刻表を作成
バス事業者がデータを整備し、利用者がマイ時刻表が作れるアプリを提供

### 公営バス向け簡単操作の予約／運行管理／乗客数カウントシステム 高知工科大学　重山さん(の代打の方)

市営バスが停留所ごとの利用者数を把握していなかった
乗務員さんが記録できるカウントアプリを作成
運行が終わったら自動でサーバに送信し、スプレッドシートに整理される

### バス音声広告実験について　 東京大学・ケイエムアドシステム共同 伊藤先生

今この瞬間に合わせた広告が流れるシステムを作成
Webアプリから広告を入稿する仕組みを導入した

**Comment by asahina820** [comment]
## **情報提供　司会：諸星賢治**

### 太田幸雄（ヴァル研究所）(仮)CPから見たBusdata

GTFSはコンピュータで扱いやすい
GTFSデータを評価するためのチェックツールがある

### 羽田野湧太「３行貼り付けるだけでバスの時刻表を埋め込めるタグ（APIとライブラリとタグ）」

時刻表って見づらい、、
乳頭温泉にいくのにGoogleマップで検索しても出てこない、、
Web上でタグを貼り付ければWebページ上で経路を見ることができる仕組みを開発

### 齋藤悠宇「GTFS-RTを活用した車窓Grapherで車窓をもっと好きになる」

車窓に広がるランドマークを通知するアプリ
GTFS-RTを用いてバスの位置を取得
追跡したいバスを選ぶとランドマークの通知が来る「まもなくスカイツリーが見えます！」
直近に通るメッシュにランドマークが含まれていれば通知する仕組み。PLATEAUデータも活用

### 岡崎峻二郎（ジオ・マーク）(仮)都バスマップ
スーパー路線図の紹介
デジタル化できなかった路線図を地図で見えるようにした

### 清水庸一郎（富士フイルムイメージングシステムズ）「バス停サイネージへのGTFS活用事例（前橋駅）」

群馬県前橋市でのバス停ソリューション
オープンデータを使ってデータ更新
バス停デジタルサイネージに広告も導入

**Comment by asahina820** [comment]
## **災害時情報（16:35〜） 司会：伊藤昌毅**

### 川原尚人「能登半島地震におけるオープンデータの価値」

災害が起きたら、平時にはないデータを再度集める！

### 塩士圭介（日本海コンサルタント） 「能登地域公共交通情報提供ページ」について

被災後、徐々に交通も復旧。公共交通状況サイトを立ち上げた
現状の復旧マップを掲示したり、特急バスの臨時便の情報共有
各事業者の発信する復旧情報を一覧にして発信、ついでに公共交通機関へ誘導して道路渋滞を緩和させたい

**Comment by asahina820** [comment]
## **公共データ・可視化（16:50～）司会：西沢明**

### にゃんこそば＠データ可視化 「大都市交通センサスでわかる 都市・交通の今」
大都市交通センサスが面白い。三代都市圏を対象に鉄道、バスの利用状況が見れる
他のオープンデータと結合することで、地図上で可視化、分析が可能
課題としては名寄せが難しいこと、データ間で事業者名や路線名が異なる、ツやケなど文字が異なる、などがある

### 武林雅衛（国土交通省）「国土数値情報交通データ」

国土数値情報について紹介

### 伊藤昌毅（東京大学）「交通データを使った授業について」

とにかく手を動かしてやってもらう授業をしている
大規模交通センサスも国土数値情報も地理院地図も使う！
PostGISも使っている、SQLで処理
自転車や経路検索、交通シミュレーションなども
東大に入るか、科目等履修生で受けてみては？

### 太田恒平（トラフィックブレイン）「GTFS-GOで広がる運行頻度図」
MIERUNE作のQGISプラグイン、GTFS-GOの紹介

**Comment by asahina820** [comment]
色々な話を聞いていて思ったことです。
- Googleマップの影響力はやはり高い
  - GTFSデータをGoogleMapsに登録したことで、バスの利用が増えた事例がいくつかあった
- GTFSデータを作るのは大変そう
  - 自治体、バス事業者任せになっている。何とか支援できないか？
- 各社、個人から可視化、分析ツールが出てきている
  - オープンデータだからこその利点。
- バスの利用ハードルを下げるために色々頑張っている。ビジネスチャンスかも
  - 観光客など初めて乗る人にも使いやすい仕組みはまだまだ検討の余地がありそう

**Comment by albertkun** [comment]
おおお！すごくノートですね！！！ちなみにｇｔｆｓデータをもっと使えたらこの英語のチュートリアは良かったと思う！

- https://medium.com/swlh/python-for-transit-get-the-most-out-of-your-gtfs-with-this-python-package-44d0b732f657

**Comment by asahina820** [comment]
> the outputs of these functions are GeoDataFrames, mostly intended to be visualized on maps but they are also susceptible to be analyzed in non-spatial plots.

GeoDataFrame! It's useful for GIS&Python engineer😆

**Comment by albertkun** [comment]
Yes and if you want all it in docker image (geopandas, postgresql, pandas) it is in this image I made :)

- https://github.com/LACMTA/geodb-image/

**Comment by hfu** [comment]
@asahina820 @albertkun 素晴らしい！ありがとうございます！！
